common/src/thrift/raw_table_master.thrift

```diff
@@ -19,7 +19,8 @@ service RawTableMasterService {
   /*
    * Creates a raw table.
    */
-  i64 createRawTable(1: string path, 2: i32 columns, 3: binary metadata)
+  i64 createRawTable(4: common.RpcOptions rpcOptions, 1: string path, 2: i32 columns, 
```

Comment:
Since we are okay breaking backwards compatibility right now, I suggest the `rpcOptions` argument is "keyed" with `1`.